base/gfspclient/gater.go

```diff
@@ -65,6 +70,34 @@ func (s *GfSpClient) ReplicatePieceToSecondary(
 	return nil
 }
 
+func (s *GfSpClient) GetPieceFromSecondary(ctx context.Context, endpoint string, task coretask.RecoveryPieceTask) (io.ReadCloser, error) {
```

Comment:
It is better to use "GetPieceFromECChunks" instead of "GetPieceFromSecondary".

Reply:
fixed